install/jaxws/docs/JAXWS3.0-ReleaseNotes.html

```diff
@@ -39,52 +39,52 @@
 </div>
 
 <hr>
 <h2><a name="kit_contents">Kit Contents</a></h2>
 
-<p>The Jakarta XML Web Services, Version 2.3 Technology Compatibility Kit (TCK) includes the following items:</p>
+<p>The Jakarta XML Web Services, Version 3.0 Technology Compatibility Kit (TCK) includes the following items:</p>
    <li><p><strong>Jakarta XML Web Services TCK Signature, API, End-to-end, WS-I Conformance, Java-to-WSDL/WSDL-to-Java Mapping and Web Services Addressing Conformance tests:</strong></p>
    <ul>
-      <li>A <strong>Signature test</strong> that checks that all of the public APIs are supported/defined as specified in the Jakarta XML Web Services Version 2.3 implementation that is being tested.</li>
+      <li>A <strong>Signature test</strong> that checks that all of the public APIs are supported/defined as specified in the Jakarta XML Web Services Version 3.0 implementation that is being tested.</li>
       <li><strong>API tests</strong> to verify the functionality of all the public APIs under <code>jakarta.xml.ws</code>, <code>jakarta.xml.ws.handler</code>, <code>jakarta.xml.ws.handler.soap</code>, <code>jakarta.xml.ws.http</code>, <code>jakarta.xml.ws.spi</code>, <code>jakarta.xml.ws.spi.http</code>, <code>jakarta.xml.ws.soap</code>, and <code>jakarta.xml.ws.wsaddressing</code>.</li>
       <li><strong>End-to-end tests</strong> that demonstrate marshalling/unmarshalling of all XML supported data types when going from WSDL-to-Java and all Jakarta XML Web Services supported Java data types when going from Java-to-WSDL as Jakarta XML Web Services SOAP Messages both through synchronous (request/response) and asynchronous (request/response) and (one-way) RPC invocations. All client programming model invocation methods (Stub-based, Dynamic Proxy, and Dispatch interface) are tested.</li>
       <li> <strong>WS-I Conformance Tests</strong> that test and check for conformance to the WS-I Basic Profile 1.1, Attachment Profile 1.0, and Simple Soap Binding Profile 1.0 conformance specifications.</li>
       <li><strong>Java-to-WSDL/WSDL-to-Java Mapping Tests</strong> that test and check for conformance to the Java-to-WSDL and WSDL-to-Java mappings.</li>
       <li><strong>Web Services Addressing Conformance Tests</strong> that test and check for conformance to the Web Services Addressing 1.0 - Core, Web Services Addressing 1.0 - Metadata, and Web Services Addressing 1.0 - SOAP Binding conformance specifications.
         <hr>
-        
+
         <h2><a name="platform">Platform Notes</a></h2>
-        
+
         <p>The Jakarta XML Web Services TCK tests have been built and tested with the the Compatible Implementation (CI) Eclipse GlassFish Server 5.1 running on the Java(tm) Standard Edition 8 (Java SE 8) platforms. If you are using a different Web server, you will need to make adjustments to the procedures outlined in this document.</p>
-        
+
         <p>The Jakarta XML Web Services TCK tests run on the following platforms:</p>
-        
+
         <ul>
```

Comment:
The Platform list looks out of date, is it useful to update that or remove it?

Reply:
We have been running the TCKs on Linux, so I think the platforms list should reflect that generically and also remove Windows 10, since we are not testing on Windows 10 (https://ci.eclipse.org/jakartaee-tck is using Linux). 

```
        <li>Oracle Linux 7.1</li>
        <li>Windows 10</li>
```

We should have:
```
        <li>Linux</li>
```

Reply:
I believe we have been using Centos 7. Not sure if that's an issue but that's what the CI jobs use, I believe.

Reply:
Correct, we have been using Centos 7.

I asked on the JakartaEE-TCK for feedback on using generic reference to OS, like Linux versus the distribution name.  

I'm not really against using an exact distribution + version, just wondering if we can make the TCK doc more generic with regard to OS, so there is less to change for each release.

Reply:
Yes, there shall be CentOS 7. I missed this one.

Reply:
Regarding something more common ...theoretically it should work on any UNIX platform that can run Java SE 8. But I would keep this list to contain only platforms that were tested.
If you have time and want to play with this, it would be nice to verify RHEL and OL (all RHEL/OL/CentOS is almost the same based on RH) and maybe some free platforms which may differ a bit.

Reply:
:+1: on keeping CentOS 7

I probably won't have time to verify JAX-WS on RHEL.